job-dsls/jobs/compile_downstream_build.groovy

```diff
@@ -18,10 +18,6 @@ def final DEFAULTS = [
                 "**/target/*.log",
                 "**/target/testStatusListener*",
                 "**/target/screenshots/**",
-                "**/target/kie-wb*wildfly*.war",
```

Comment:
Removing archiving of all wars from compile downstream jobs. It doesn't make sense to archive any wars in compile downstream jobs, because gwt compilation is skipped everywhere.